src/components/ui/collapse/Collapse.vue

```diff
@@ -8,8 +8,8 @@
             <div slot="body">
               <div class="va-row">
                 <div class="flex md6">
-                  <div class="collapse-page__content">
-                    <h4 class="collapse-page__content__title">February 2018</h4>
+                  <div class="collapse-page__content pa-4">
```

Comment:
You can probably remove `collapse-page__*` styles. They're informative, but serve no styling purpose.